frontend/src/component/archive/ArchiveTable/ArchiveTable.tsx

```diff
@@ -222,16 +224,24 @@ export const ArchiveTable = ({
         useSortBy
     );
 
-    useEffect(() => {
-        const hiddenColumns = ['description'];
-        if (isMediumScreen) {
-            hiddenColumns.push('lastSeenAt', 'status');
-        }
-        if (isSmallScreen) {
-            hiddenColumns.push('type', 'createdAt');
-        }
-        setHiddenColumns(hiddenColumns);
-    }, [setHiddenColumns, isSmallScreen, isMediumScreen]);
+    useConditionallyHiddenColumns(
+        [
+            {
+                condition: true,
+                columns: ['description'],
+            },
+            {
+                condition: isSmallScreen,
+                columns: ['type', 'createdAt'],
+            },
+            {
+                condition: isMediumScreen,
+                columns: ['lastSeenAt', 'stale'],
```

Comment:
These columns were not being hidden correctly because their id's didn't match, so I fixed it.